src/core/types/addons.js

```diff
@@ -122,6 +141,15 @@ export type PartialExternalAddonType = {|
  */
 export type AddonType = {|
   ...ExternalAddonType,
+  // normalized l10n fields
```

Comment:
This is great but not true in the current patch. The assumption was that `selectLocalizedContent()` return type would be `string | null`. I think it's what we want and I explained why in a previous comment (so let's keep these changes here and update the selector a bit)